doc/user/flycheck-versus-flymake.rst

```diff
@@ -131,90 +121,41 @@ explicit agreement that your contributions will be under the GPL.
 Enabling syntax checking
 ------------------------
 
-**Flymake** is not enabled automatically for supported languages.  It must be be
-enabled for each mode individually and **carefully**, because it does not deal
-well with unavailable syntax checker tools.  In a GUI frame, it signals errors
-in GUI dialogs.  In a TTY frame, it does not signal any error at all, but
-instead silently hangs.  The same occurs when a syntax checker tool becomes
-unavailable after Flymake Mode is enabled (for instance, because the underlying
-tool was uninstalled).
-
-.. figure:: /images/flymake-error.png
-   :align: center
-
-   Flymake showing a GUI dialog to inform that a syntax checker tool is not
-   available
-
-The third-party library flymake-easy_ provides an alternate way to enable
-Flymake Mode, which gracefully handles unavailable syntax checkers.  It does not
-check whether the tool still exists before a syntax check, though, and thus does
-still exposes above behavior when a tool becomes unavailable after the mode was
-enabled.
+**Flymake** is not enabled automatically for supported languages.  It must be
+enabled for each mode individually.
```

Comment:
"Or you can just enable it for every prog-mode buffer and it'll detect automatically if there's a checker available, otherwise, it won't bug you much..."

Reply:
Sure, I'll add that.